_api-reference/security/index.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: default
+title: Security APIs
+nav_order: 50
+has_children: true
+redirect_from:
+  - /api-reference/security-api/
+---
+
+# Security APIs
+
+The Security plugin provides numerous REST APIs for managing its resources. These APIs are similar to the OpenSearch REST APIs and consist of HTTP requests that include the resource path, HTTP method (GET, PUT, POST, DELETE), request body, and output response fields. 
+
+All Security APIs use the base path of `_plugins/_security/` followed by the specific path for each operation. For example, the path for the Role API would be `_plugins/_security/api/roles/`.
```

Comment:
I can't find an API called either the "Role API" or the "Roles API" in our documentation. Can you point me to an instance?

Reply:
This is a missing API. There's about 6 more security sections that still need to be added. I can point this to an API in this section, if you prefer.

Reply:
Yeah, let's point to an existing API 😄.